src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Metadata.php

```diff
@@ -128,6 +128,11 @@ class Metadata
      */
     public $organizationTypeAttribute;
 
+    /**
+     * @var Attribute
+     */
+    public $organizationUnitAttribute;
+
```

Comment:
Please scrap this from the commit. It's a patch to work with the older Manage instance in our container

Reply:
Ok